clearpath_control/config/a300/teleop_joy.yaml

```diff
@@ -0,0 +1,12 @@
+# A300 default Joy Teleop velocity limits
+teleop_twist_joy_node:
+  ros__parameters:
+    # Regular
+    scale_linear.x: 0.4
+    scale_linear.y: 0.4
```

Comment:
Have we validated that these gains work with the mecanum wheels on A300, or are these a placeholder until the hardware comes in? It's fine either way, just asking for the sake of transparency.

Reply:
Tested with robot on blocks, may need to be adjusted once the wheels come in. I've just kept them the same as linear X to match what we have for Dingo and Ridgeback.